sfge/src/main/java/com/salesforce/graph/symbols/apex/schema/DescribeSObjectResultFactory.java

```diff
@@ -15,7 +17,22 @@ public final class DescribeSObjectResultFactory {
             vertex -> {
                 List<String> chainedNames = vertex.getChainedNames();
 
-                if (chainedNames.size() == 2) {
+                if (chainedNames.size() == 1) {
+                    // SObjectType.Account
```

Comment:
To be sure I understand, this is when you reference something like `SObjectType.Account` without using the `Schema` namespace first? If so, could a comment be added to explain how this differs from the case at line 35?

Reply:
Right, that's the difference. Will add more information to the code.